frontend/packages/ceph-storage-plugin/src/components/disk-inventory/ocs-disks-list.tsx

```diff
@@ -181,9 +184,8 @@ const OCSDisksList: React.FC<TableProps> = React.memo((props) => {
 
     if (tiLoaded && !tiLoadError && tiData.length) {
       const newData: OCSDiskList = tiData.reduce((data, ti) => {
-        const disk = ti.metadata.annotations?.disk;
-        const osd = ti.metadata.annotations?.osd;
-        if (osd && disk) {
+        const { disk, node, osd } = getAnnotations(ti) || {};
+        if (osd && disk && node === nodeName && ocsState.metricsMap?.[disk]?.osd === osd) {
```

Comment:
oh ok. it showing status from some other nodes TemplateInstance, you need some property to identify TemplateInstance from other nodes to avoid a collision. that is why you nodeName annotation in TemplateInstance.

Reply:
Yes!